public/pages/SplitIndex/components/SplitIndexFlyout/SplitIndexFlyout.tsx

```diff
@@ -52,6 +52,7 @@ export default class SplitIndexFlyout extends Component<SplitIndexComponentProps
       {
         rowProps: {
           label: "Target Index Name",
+          helpText: `Must be in lowercase letters. Cannot begin with underscores or hyphens. Spaces, commas, and characters :, ", *, +, /, \, |, ?, #, > are not allowed.`,
```

Comment:
can we have this help text in common place and reuse for create/shrink index page?

Reply:
Added index name and replica help text to constants.ts. shard number help varies depending on operation, for example, split doesn't want to show message as default shard number is 1. So didn't add one common help text for shard number.